Commands/Migrate.php

```diff
@@ -110,7 +100,7 @@ public function logMessage($message)
         } else {
             $now = Date::now()->getDatetime();
         }
-        $this->output->writeln($message . ' at ' . $now);
+        $this->getOutput()->writeln($message . ' at ' . $now);
```

Comment:
should we use below ?
```suggestion
        $output->writeln($message . ' at ' . $now);
```

Reply:
With Matomo 5, the `output` property of `ConsoleCommand` is private and `$output` isn't being passed into that method. Are you suggesting that I add it as an argument for the `logMessage` method?

Reply:
sorry, I missed that was a separate function